src/components/Incorporation/ListShareClass.vue

```diff
@@ -20,7 +20,7 @@
         disable-sort
         hide-default-footer
       >
-        <template v-slot:item="row" class="share-data-table">
```

Comment:
Class is not valid on a template element. I searched the code and this class wasn't referenced anywhere.